masonry/examples/calc_masonry.rs

```diff
@@ -145,20 +145,25 @@ impl Widget for CalcButton {
         match event {
             PointerEvent::PointerDown(_, _) => {
                 if !ctx.is_disabled() {
-                    ctx.get_mut(&mut self.inner)
-                        .set_background(self.active_color);
+                    let color = self.active_color;
+                    ctx.mutate_later(&mut self.inner, move |mut inner| {
```

Comment:
I think that this should have a comment, explaining why we can't access the background/box directly.